test/unit/browserify-builder.js

```diff
@@ -22,7 +22,8 @@ test('browserify builder', function (done) {
   var builder = builderBrowserify(files, config)
 
   builder(function (_, src) {
-    assert.ok(src.indexOf("console.log('qux');") !== -1)
-    done()
+    t.notEqual(src.indexOf("console.log('qux');"), -1)
```

Comment:
Nit for future PR: I personally find a positive assertion easier to read.

E.g. `t.ok(src.indexOf("console.log('qux');") >= 0)`

Reply:
Sure. We also need much more tests in general.